sim_pipeline/para_distribution.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+class GaussianMixtureModel:
```

Comment:
can you make a test function of this class and their functions?
Also, perhaps make a separate folder within sim-pipeline called ParamDistributions and make a file called gaussian_mixture_model.py? (also add an __init__.py file to the folder). Then the code structure is better organized